### ISSUE TITLE
Process queue config

### ISSUE THREAD

**Issue by grvoyt** [hhxsv5/laravel-s#478]
1. Your software version (Screenshot of your startup)
    
    | Software | Version |
    | --------- | --------- |
    | PHP | 8.3 |
    | Swoole | 5+ |
    | Laravel| 10+ |

2. Detail description about this issue(error/log)

    Mistake in code.
Hhxsv5\LaravelS\Swoole\Process\CustomProcessTrait(103)

$msgKey = isset($config['msg_key']) ? $config['msg_key'] : 0;

As you declare in doc, must be:

$msgKey = isset($config['queue']['msg_key']) ? $config['queue']['msg_key'] : 0;

in Doc

```
'queue'    => [ 
      'msg_key'  => 0, 
      'mode'     => 2,    
      'capacity' => 8192, 
],
```

**Comment by hhxsv5** [comment]
Thanks for your feedback.

**Comment by grvoyt** [comment]
when is the release of the certificates approximately?

> Thanks for your feedback.

**Comment by hhxsv5** [comment]
Released [v3.8.1](https://github.com/hhxsv5/laravel-s/releases/tag/v3.8.1).
